plugins/inputs/knx_listener/knx_helpers.go

```diff
@@ -0,0 +1,205 @@
+	d_5004 := dpt.DPT_5004(25)
+	data = append(data, generate_event("5/0/4", &d_5004))
+
+	// DPT 9.xxx
+	d_9001 := dpt.DPT_9001(18.56)
+	data = append(data, generate_event("9/0/1", &d_9001))
+	d_9004 := dpt.DPT_9004(243.9)
+	data = append(data, generate_event("9/0/4", &d_9004))
+	d_9005 := dpt.DPT_9005(12.01)
+	data = append(data, generate_event("9/0/5", &d_9005))
+	d_9007 := dpt.DPT_9007(59.32)
+	data = append(data, generate_event("9/0/7", &d_9007))
+
+	// DPT 12.xxx
+	d_12001 := dpt.DPT_12001(1234567)
+	data = append(data, generate_event("12/0/1", &d_12001))
+
+	// DPT 13.xxx
+	d_13001 := dpt.DPT_13001(13001)
+	data = append(data, generate_event("13/0/1", &d_13001))
+	d_13002 := dpt.DPT_13002(13002)
+	data = append(data, generate_event("13/0/2", &d_13002))
+	d_13010 := dpt.DPT_13010(130010)
+	data = append(data, generate_event("13/1/0", &d_13010))
+	d_13011 := dpt.DPT_13011(130011)
+	data = append(data, generate_event("13/1/1", &d_13011))
+	d_13012 := dpt.DPT_13012(130012)
+	data = append(data, generate_event("13/1/2", &d_13012))
+	d_13013 := dpt.DPT_13013(130013)
+	data = append(data, generate_event("13/1/3", &d_13013))
+	d_13014 := dpt.DPT_13014(130014)
+	data = append(data, generate_event("13/1/4", &d_13014))
+	d_13015 := dpt.DPT_13015(130015)
+	data = append(data, generate_event("13/1/5", &d_13015))
+
+	return data
+}
+
+func GetDatapointType(name string) (d dpt.DatapointValue, err error) {
+	switch name {
+	case "1.001":
+		d, err = new(dpt.DPT_1001), nil
+	case "1.002":
+		d, err = new(dpt.DPT_1002), nil
+	case "1.003":
+		d, err = new(dpt.DPT_1003), nil
+	case "1.009":
+		d, err = new(dpt.DPT_1009), nil
+	case "1.010":
+		d, err = new(dpt.DPT_1010), nil
+	case "5.001":
+		d, err = new(dpt.DPT_5001), nil
+	case "5.003":
+		d, err = new(dpt.DPT_5003), nil
+	case "5.004":
+		d, err = new(dpt.DPT_5004), nil
+	case "9.001":
+		d, err = new(dpt.DPT_9001), nil
+	case "9.004":
+		d, err = new(dpt.DPT_9004), nil
+	case "9.005":
+		d, err = new(dpt.DPT_9005), nil
+	case "9.007":
+		d, err = new(dpt.DPT_9007), nil
+	case "12.001":
+		d, err = new(dpt.DPT_12001), nil
+	case "13.001":
+		d, err = new(dpt.DPT_13001), nil
+	case "13.002":
+		d, err = new(dpt.DPT_13002), nil
+	case "13.010":
+		d, err = new(dpt.DPT_13010), nil
+	case "13.011":
+		d, err = new(dpt.DPT_13011), nil
+	case "13.012":
+		d, err = new(dpt.DPT_13012), nil
+	case "13.013":
+		d, err = new(dpt.DPT_13013), nil
+	case "13.014":
+		d, err = new(dpt.DPT_13014), nil
+	case "13.015":
+		d, err = new(dpt.DPT_13015), nil
+	default:
+		d, err = nil, fmt.Errorf("unknown data-point type: %s", name)
+	}
+	return
+}
+
+func GetBasicDatapointValue(d dpt.DatapointValue) (v interface{}, err error) {
+	vi := reflect.Indirect(reflect.ValueOf(d))
+	switch vi.Kind() {
+	case reflect.Bool:
+		v, err = vi.Bool(), nil
+	case reflect.Int:
+		fallthrough
+	case reflect.Int8:
+		fallthrough
+	case reflect.Int16:
+		fallthrough
+	case reflect.Int32:
+		fallthrough
+	case reflect.Int64:
```

Comment:
FYI, you can do: 
`case reflect.Int, reflect.Int8, reflect.Int16, reflect.Int32, reflect.Int64:` all on one line.

Reply:
Done